java-source/src/main/java/com/example/client/ExampleClientRPC.java

```diff
@@ -27,7 +27,7 @@ public static void main(String[] args) throws ActiveMQException, InterruptedExce
 
         final Logger logger = LoggerFactory.getLogger(ExampleClientRPC.class);
         final HostAndPort nodeAddress = HostAndPort.fromString(args[0]);
-        final CordaRPCClient client = new CordaRPCClient(nodeAddress, null, null);
+        final CordaRPCClient client = new CordaRPCClient(nodeAddress, ConfigUtilities.configureTestSSL(), null);
```

Comment:
Not in Java without changing the how the Corda class declaration is written, as it complains bitterly right now.